data/json/items/ammo/223.json

```
@@ -127,6 +127,15 @@
     "ascii_picture": "556_incendiary",
     "extend": { "effects": [ "INCENDIARY" ] }
   },
+  {
+    "id": "556_ap",
+    "copy-from": "556",
+    "type": "AMMO",
+    "name": { "str_sp": "5.56 NATO M995" },
+    "description": "An M995 armor piercing round with dense tungsten core and sharp, paint in black tip.  Higher material penetration in exchange of lower overall damage.",
```

```suggestion
    "description": "An M995 armor-piercing round using a 52 gr. projectile, featuring a black-tipped, tungsten carbide penetrator over an aluminum slug.  Higher material penetration in exchange of lower overall damage.",
```
The other descriptions have bullet weights in them 👉👈